pandas/core/window.py

```diff
@@ -42,7 +43,7 @@
 class _Window(PandasObject, SelectionMixin):
     _attributes = ['window', 'min_periods', 'center', 'win_type',
                    'axis', 'on', 'closed']
-    exclusions = set()
+    exclusions = set()  # type: Set
```

Comment:
Can you annotate what the set should hold?

Reply:
I _think_ it's `str`s, but I'm not sure.